pulsar-client/src/main/java/org/apache/pulsar/client/impl/PulsarChannelInitializer.java

```diff
@@ -72,7 +72,10 @@ public PulsarChannelInitializer(ClientConfigurationData conf, Supplier<ClientCnx
         if (tlsEnabled) {
             if (tlsEnabledWithKeyStore) {
                 AuthenticationDataProvider authData1 = conf.getAuthentication().getAuthData();
-
+                if (conf.getTlsTrustStorePath() == null) {
+                    throw new RuntimeException("Failed to create TLS context, Due to empty "
```

Comment:
```suggestion
                    throw new RuntimeException("Failed to create TLS context, due to empty "
```

Reply:
I think it's better to use like "Failed to create TLS context, the tlsTrustStorePath need to be configured if useKeyStoreTls enabled" ?

Reply:
Hi, @Jason918.
I agree with @codelipenghui.